src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -118,538 +127,644 @@ export class JailAddComponent implements OnInit {
+    },
+    {
       type: 'checkbox',
       name: 'exec_clean',
-      placeholder: T('exec_clean'),
+      placeholder: T('exec.clean'),
       tooltip: T('Run commands in a clean environment. The current\
- environment is discarded except for HOME, SHELL, TERM and USER. HOME\
- and SHELL are set to the target login default values. USER is set to\
- the target login. TERM is imported from the current environment. The\
- environment variables from the login class capability database for the\
- target login are also set.'),
-    }, {
+                 environment is discarded except for $HOME, $SHELL, \
+                 $TERM, and $USER. <br>\
+                 $HOME and $SHELL are set to the target login. $USER is \
+                 set to the target login. $TERM is imported from the \
+                 current environment. The environment variables from the \
+                 login class capability database for the target login \
+                 are also set.'),
+    },
+    {
       type: 'input',
       name: 'exec_timeout',
-      placeholder: T('exec_timeout'),
-      tooltip: T('Define the maximum amount of time in seconds to wait for\
- a command to complete. If a command is still running after the allotted\
- time, the jail will be terminated.'),
-    }, {
+      placeholder: T('exec.timeout'),
+      tooltip: T('Maximum amount of time in seconds to wait for a \
+                  command to complete. If a command is still running \
+                  after the allotted time, the jail will not be \
+                  created or removed.'),
+    },
+    {
       type: 'input',
       name: 'stop_timeout',
-      placeholder: T('stop_timeout'),
-      tooltip: T('Define the maximum amount of time in seconds to wait for\
- the jail processes to exit after sending a SIGTERM signal. This happens\
- after any <b>exec_stop</b> commands are complete. After the defined time,\
- the jail is removed, killing any remaining processes. If this is set to\
- <i>0</i>, no SIGTERM is sent and the jail is immediately removed.'),
-    }, {
+      placeholder: T('stop.timeout'),
+      tooltip: T('Maximum amount of time in seconds to wait for jail \
+                 processes to exit after sending a SIGTERM signal. The \
+                 timeout starts after <b>exec.stop</b> commands have \
+                 completed. When time is up the jail is removed, killing \
+                 any remaining processes. When set to <i>0</i>, no \
+                 SIGTERM is sent and the jail is immediatley removed. \
+                 The default timeout is 10 seconds.'),
+    },
+    {
       type: 'input',
       name: 'exec_jail_user',
-      placeholder: T('exec_jail_user'),
-      tooltip: T('Enter either <i>root</i> or a valid <i>username</i>. In\
- the jail environment, commands run as this defined user.'),
-    }, {
+      placeholder: T('exec.jail_user'),
+      tooltip: T('The user to run commands as in the jail environment. \
```

Comment:
"as in the jail environment" - I'm not sure if the "as" is redundant or if something more specific is being communicated.

Reply:
This refers to the user account to execute commands as. Waiting to discuss with Warren.